src/boss/azure_worker.go

```diff
@@ -0,0 +1,214 @@
+package boss
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/user"
+	"sync"
+	"time"
+)
+
+type AzureWorkerPool struct {
+	workerNum int
+	workers   *map[string]*AzureWorker
+	nextId    int
+}
+
+type AzureWorker struct {
+	pool         *AzureWorkerPool
+	workerId     string
+	configPosit  int
+	diskName     string
+	vnetName     string
+	subnetName   string
+	nsgName      string
+	nicName      string
+	publicIPName string
+	privateAddr  string
+	publicAddr   string
+}
+
+func NewAzureWorkerPool() *WorkerPool {
+	conf, err := ReadAzureConfig()
+	if err != nil {
+		log.Fatalln(err)
+	}
+	num := conf.Resource_groups.Rgroup[0].Numvm
```

Comment:
Do we support multiple resource groups?  If not, we should probably check the length and return an error if it is not 1.